docs/navigate.rst

```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
 2. In the top right of the Visibility tab, click :guilabel:`Refresh`.
 
 The contents of the visibility table are refreshed.
+
+
+
+Configure Telemetry
+-------------------
+
+1. To view the Virtual Server statistics on the Visibility VIP Dashboard, Telemetry plugin version 1.17 or higher needs to be installed on the BIG-IP device.
+
+2. Follow the installation steps from https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/installation.html
+
+3. Configure a default pull consumer. For example: https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/pull-consumers.html#pull
+
+4. Recommended pull consumer configuration: For a scaled environment it is highly recommended to retrieve just the Pool and Virtual Server stats from the BIG-IP. One way filter the VIP and Pool statistics is to create a default pull consumer configuration via  a POST request to: https://<BIG_IP>/mgmt/shared/telemetry/declare
```

Comment:
For a scaled environment, we strongly recommend you retrieve just the Pool...

One way to filter the VIP... <missing the word 'to'>